executor/insert_common.go

```diff
@@ -716,23 +716,23 @@ func (e *InsertValues) lazyAdjustAutoIncrementDatum(ctx context.Context, rows []
 				i++
 				cnt++
 			}
-			// Alloc batch N consecutive (min, max] autoIDs.
-			// max value can be derived from adding one for cnt times.
-			min, _, err := table.AllocBatchAutoIncrementValue(ctx, e.Table, e.ctx, cnt)
+			// Alloc batch N consecutive autoIDs, returning min and increment.
```

Comment:
`returning min and increment.` I think this comment is redundant, the code below this already said so.

Reply:
nice, addressed.